pkg/cmd/addon/install.go

```diff
@@ -216,8 +256,8 @@ It will automatically skip version checks, which may result in the cluster not r
 	return err
 }
 
-// Run will apply the addon.yaml to K8s
-func (o *installOption) Run() error {
+// Run will apply the addon.yaml to K8s and register the cluster chart
```

Comment:
This function only creates the addon yaml, does not register the cluster chart. Maybe call RegisterClusterChart in this function or modify this comment.